infrastructure/variables.tf

```diff
@@ -51,10 +51,6 @@ variable "use_uk_db" {
 // S2S
 ////////////////////////////////
 
-variable "s2s_url" {
```

Comment:
Consequently, this variable needs to be removed from `aat.tfvars`, `demo.tfvars`, `prod.tfvars` and `preview.tfvars`.